mods/d2k/rules/defaults.yaml

```diff
@@ -225,6 +227,11 @@
 			Rough: 80
 	Voiced:
 		VoiceSet: InfantryVoice
+	WithParachute:
+		UpgradeTypes: parachute
+		UpgradeMinEnabledLevel: 1
+		ShadowImage: parach-shadow
+		ShadowSequence: idle
```

Comment:
I tend to go XCC Mixer (being part of XCC Utilities) from http://xhp.xwis.net/utilities/ due to that being practically the swiss knife WW modding tool. It has some bugs - only read pals within mixes, TS and RA2 folders are read from registry - but people can easily create a mix and feed it's folder as TD or RA1 install folder instead. It works under Wine.